Foundation/NSSwiftRuntime.swift

```diff
@@ -81,12 +81,12 @@ internal func _CFZeroUnsafeIvars<T>(_ arg: inout T) {
 }
 
 @_cdecl("__CFSwiftGetBaseClass")
-internal func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
+public func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
     return unsafeBitCast(__NSCFType.self, to:UnsafeRawPointer.self)
 }
 
 @_cdecl("__CFInitializeSwift")
-internal func __CFInitializeSwift() {
+public func __CFInitializeSwift() {
```

Comment:
This may be an Xcode bug? (very skeptical of claiming that without a very clear reduced case). I will investigate more and let you know.